pkg/validation/spec/gnostic.go

```diff
@@ -1263,6 +1263,8 @@ func (k *Schema) FromGnostic(g *openapi_v2.Schema) (ok bool, err error) {
 			k.AdditionalProperties.Allows = g.AdditionalProperties.GetBoolean()
 		} else {
 			k.AdditionalProperties.Schema = &Schema{}
+			k.AdditionalProperties.Allows = true
```

Comment:
Why is this always true here?

Reply:
This mirrors the unmarshalling behavior:

https://github.com/kubernetes/kube-openapi/blob/fa38f90500eab05fc36808c4f35fb45e5297fca6/pkg/validation/spec/swagger.go#L145-L156

We want the fuzzer to generate valid OpenAPI objects that are capable of round-tripping. If `Schema` is non-nil and `Allows` is false, that is an impossible object to round-trip.

Reply:
OK, I found it hard to figure out why, but that sounds somewhat reasonable to me. I think the swagger decoder is wrong to set it to true, but that's the behavior so let's keep it as is.